ocs_ci/ocs/resources/pvc.py

```diff
@@ -182,7 +182,7 @@ def resize_pvc(self, new_size, verify=False):
 
         # Modify size of PVC
         assert self.ocp.patch(
-            resource_name=self.name, params=patch_param
+            resource_name=self.name, params=patch_param, format_type="merge"
```

Comment:
We are not seeing any issues with the resize pvc without the pass of the format_type. Do you see any issues in your case?